lib/gruf/controllers/request.rb

```diff
@@ -67,6 +70,20 @@ def service_key
         @service.name.underscore.tr('/', '.').gsub('.service', '')
       end
 
+      ##
+      # @return [Class]
```

Comment:
To nit pick a little do you mean `@return [Object]`. Or is `[Class]` the preferred ruby way?

Reply:
So `Class` _is_ an `Object` in Ruby, which is why I'm specific here about the actual class being returned.

Fwiw, the ancestry chain is: `Class -> Module -> Object -> Kernel -> BasicObject`

Reply:
That makes sense. Thanks! :shipit: